webapp/portlet/src/main/webapp/vue-apps/common/initComponents.js

```diff
@@ -189,4 +191,50 @@ Vue.directive('cacheable', {
       vnode.componentInstance.$root.$emit('application-mount', true);
     }, 10000);
   },
-});
\ No newline at end of file
+});
+
+Vue.directive('draggable', {
+  bind(el, binding) {
+    let counter = 0;
+    const dropZoneBindingValue = binding?.value;
+    const dropZoneParentElement = dropZoneBindingValue?.parentElementId ? document.getElementById(dropZoneBindingValue.parentElementId) : el;
+    const attachmentEnabled = dropZoneBindingValue?.attachmentEnabled;
```

Comment:
Disabled drag effect independently from `attachmentEnabled` flag, use more generic name

```suggestion
    const enabled = dropZoneBindingValue.enabled;
```

Reply:
done